.run/build.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/bash -ex
+
+AGENT_ROOT_DIRECTORY=$(git rev-parse --show-toplevel)
+
+if [[ -z $AGENT_SKIP_VENV ]]; then
+  # This script assumes that a python venv is located under the root directory
+  AGENT_VENV_DIR=venv
```

Comment:
I think that's a fare assumption but worth making it configurable like

```bash
AGENT_VENV_DIR=${AGENT_VENV_DIR:-venv}
```

Reply:
that way people with different venv location can use their preferred location of venv